src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -771,6 +771,11 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                     'System.Management.Automation.InvocationInfo'
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Get-Error tests.Get-Error adds ExceptionType for Exceptions
```powershell
Expected case sensitive like wildcard '*System.Net.Internals.SocketExceptionFactory+ExtendedSocketException*' to match '
Exception             : 
    ExceptionType  : System.Management.Automation.MethodInvocationException
    ErrorRecord    : 
        Exception             : 
            ExceptionType : System.Management.Automation.ParentContainsErrorRecordException
            Message       : Exception calling "GetHostByName" with "1" argument(s): "No such host is known."
            HResult       : -2146233087
        CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
        FullyQualifiedErrorId : SocketException
        InvocationInfo        : 
            ScriptLineNumber : 96
            OffsetInLine     : 13
            HistoryId        : -1
            ScriptName       : D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1
            Line             : [System.Net.DNS]::GetHostByName((New-Guid))
            PositionMessage  : At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1:96 char:13
                               +             [System.Net.DNS]::GetHostByName((New-Guid))
                               +             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            PSScriptRoot     : D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility
            PSCommandPath    : D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1
            CommandOrigin    : Internal
        ScriptStackTrace      : at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1: line 96
                                at Invoke-Test, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Functions\It.ps1: line 281
                                at ItImpl, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Functions\It.ps1: line 233
                                at It, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Functions\It.ps1: line 124
                                at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1: line 94
                                at DescribeImpl, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Functions\Describe.ps1: line 199
                                at Describe, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Functions\Describe.ps1: line 91
                                at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1: line 4
                                at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
                                at Invoke-Pester<End>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1123
                                at <ScriptBlock>, <No file>: line 1
    TargetSite     : 
        Name          : CheckActionPreference
        DeclaringType : System.Management.Automation.ExceptionHandlingOps
        MemberType    : Method
        Module        : System.Management.Automation.dll
    StackTrace     : 
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
    Message        : Exception calling "GetHostByName" with "1" argument(s): "No such ho
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1: line 103
103:         $out | Should -BeLikeExactly '*System.Net.Internals.SocketExceptionFactory+ExtendedSocketException*'
```